### ISSUE TITLE
How to make daily bandwidth limit

### ISSUE THREAD

**Issue by clintnetwork** [z3APA3A/3proxy#634]
Hey,

Is it possible to implement a daily bandwidth limit or something like this?

Thanks

**Comment by z3APA3A** [comment]
Probably, you mean traffic limit. See counter + conuntin/countout/countall commands.

**Comment by clintnetwork** [comment]
Yes I thought about counter, do you think its possible with this commands to setup daily limit in gigabyte?

**Comment by z3APA3A** [comment]
Yes, it's possible with counter + conuntin/countout/countall
